lms/djangoapps/courseware/tests/helpers.py

```diff
@@ -22,6 +32,72 @@ def get_request_for_user(user):
     return request
 
 
+def answer_entrance_exam_problem(course, request, problem, user=None):
```

Comment:
They are helper functions, of course.  But they are not generic enough to belong in this file.  Since they are specific to entrance exams, they can be defined in the test_entrance_exam.py module.